packages/schema/src/types/global/vite.ts

```diff
@@ -20,15 +20,16 @@ export interface ViteHot {
 
 export interface ViteGlobOptions {
   as?: string
+  eager?: boolean
+  import?: string
+  query?: string | Record<string, string | number | boolean>
+  exhaustive?: boolean
 }
 
 export interface ViteImportMeta {
   /** Vite client HMR API - see https://vitejs.dev/guide/api-hmr.html */
   readonly hot?: ViteHot
 
   /** vite glob import utility - https://vitejs.dev/guide/features.html#glob-import */
-  glob?(pattern: string, options?: ViteGlobOptions): Record<string, () => Promise<Record<string, any>>>
-
-  /** vite glob import utility - https://vitejs.dev/guide/features.html#glob-import */
-  globEager?(pattern: string, options?: ViteGlobOptions): Record<string, Record<string, any>>
+  glob (glob: string | string[], options?: ViteGlobOptions): Record<string, () => Promise<Record<string, any>>>
```

Comment:
This is meant to be optional as the declaration is also present in webpack builder.

```suggestion
  glob?(glob: string | string[], options?: ViteGlobOptions): Record<string, () => Promise<Record<string, any>>>
```